src/llmq/quorums_signing.cpp

```diff
@@ -575,7 +574,7 @@ void CSigningManager::CollectPendingRecoveredSigsToVerify(
         for (auto it = v.begin(); it != v.end();) {
             auto& recSig = *it;
 
-            Consensus::LLMQType llmqType = (Consensus::LLMQType) recSig.llmqType;
+            auto llmqType = (Consensus::LLMQType) recSig.llmqType;
```

Comment:
why not just
```suggestion
            Consensus::LLMQType llmqType = recSig.llmqType;
```
or
```suggestion
            auto llmqType = recSig.llmqType;
```

I anyway asked myself many times already why we have so many more places in the code where we cast variables from type `Consensus::LLMQType` to `Consensus::LLMQType` 😄 Doesn't make a lot sense imo unless thats some fancy practice im not aware of.

Reply:
Hmm, I'm not sure why we do that... Just know this resolves the clang warning